openmc/material.py

```diff
@@ -311,6 +327,65 @@ def remove_nuclide(self, nuclide):
         if nuclide._name in self._nuclides:
             del self._nuclides[nuclide._name]
 
+    def add_macroscopic(self, macroscopic):
```

Comment:
Since a `Material` can only have a single `Macroscopic`, why not make this a `macroscopic` property setter instead of an "adder" method?